CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Kiwix JS Contribution Guide
+
+If you have some development experience with HTML and JavaScript, we welcome Pull Requests on existing issues. Please look at this repository's Issue
+tracker, in particular those marked "good first issue". If you have found a bug, check first that we do not have an existing issue covering it, and if not, feel free to create an issue.
+
+Please follow these guidelines when contributing:
+
+- Ask to be assigned to an issue you wish to work on first (we have lots of past issues, some of which are no longer relevant or wanted);
+- We ask you only to pick issues for which you are confident that you have a solution;
+- If you are a newcomer to the repo, create a personal fork and clone it locally. Check out a new branch with a name relevant to the issue you are working on;
+- Follow the coding style of the area you are editing, including indentation, and be consistent with quotation marks and spacing;
+- Use no higher than [ECMAScript 5](https://caniuse.com/es5) - notably, do not use arrow functions or `async` functions. However, Promises *are*
+  supported via a polyfill;
+- Do not prettify code you are not working on;
+- You must test your code yourself before asking for review, like this:
+  - set up a local Web server (we recommend Node/NPM's [http-server](https://www.npmjs.com/package/http-server));
+  - serve the root directory of the repository (e.g. `http-server .`);
+  - in a browser, navigate to the URL of the main `index.html` (e.g. http://localhost:8080/www/index.html);
+  - manually test your fix in at least Firefox and Chromium (Edge or Chrome), ideally also in IE11 or in "IE Mode" in Edge;
+  - be sure to test your fix in both "JQuery" mode and "Service Worker" mode (see Configuration);
+  - run the Unit tests (see below) in at least the above browsers.
```

Comment:
@Jaifroid One thing that I just noticed and missed earlier, here in line 21 we are referring to the Unit tests' instructions in the same file which is not the case as we moved this section to a separate file. So, should I refer it back to README.md or move the Unit tests section too in this file?

Reply:
Thanks for noticing that. My inclination is to move the Unit Tests section to here, as it is only really useful information for contributors, but let's get @mossroy's advice on this.

Reply:
Sure

Reply:
I agree with @Jaifroid